doc-content/enterprise-only/installation/clustering-bc-testing-proc.adoc

```diff
@@ -0,0 +1,26 @@
+[id='clustering-bc-testing-proc']
+= Verifying the {PRODUCT} cluster
+
+After you have configured the cluster for {PRODUCT}, create an asset to verify that the installation is working.
+
+.Procedure
+. In a web browser, enter `<node-IP-address>:8080/{URL_COMPONENT_CENTRAL}`. Replace `<node-IP-address>` with the IP address of a particular node.
+. Enter the `admin` user credentials that you created during installation. The {CENTRAL} home page appears.
+//. Select *Menu* -> *Deploy* -> *Execution Servers*.
+//. Confirm that *default-kieserver* is listed under *Server Configurations*.
+. Select *Menu* -> *Design* -> *Projects*.
+. Click *Try Samples* -> *Mortgages* -> *OK*. The Assets window appears.
+. Click *Create New Asset* -> *Data Object*.
+. Enter `MyDataObject` in the *Data Object* field and click *OK*.
+. Click *Spaces* -> *myteam* -> *Mortgages* and confirm that `MyDataObject` is in the list of assets.
+. Enter the following URL, where `<node_IP_address>` is the address of a different node of the cluster:
```

Comment:
Enter this URL where? In the browser address field, I presume? Then perhaps "Open the following URL"? "Enter" can refer to entry in any field at all.